docs/.vuepress/public/scripts/fixer.js

```diff
@@ -86,6 +86,9 @@
       } else if (key === 'handsontable/base') {
         ns = 'Handsontable';
 
+      } else if (key === 'hyperformula') {
+        ns = 'HyperFormula';
+
```

Comment:
What does this fix? How can I observe it?

Reply:
This is meant to enable our JSFiddle examples to simulate `import { HyperFormula } from 'hyperformula';` while using the global `HyperFormula`.